### PR TITLE
AC-796 Updated UI for synced patient cards

### PR DESCRIPTION
**Pull request by rishabh-997**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-796

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by rishabh-997** [comment]
Screenshot_2020-07-07-10-24-00-30_0b570628ec88a4202b7916ec2df1bebf

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsRecyclerViewAdapter.java [hunk 1]
```diff
@@ -98,7 +99,18 @@ public void onBindViewHolder(@NonNull SyncedPatientsRecyclerViewAdapter.PatientV
             holder.mDisplayName.setText(patient.getName().getNameString());
         }
         if (null != patient.getGender()) {
-            holder.mGender.setText(patient.getGender());
+            if (patient.getPhoto()!=null ){
+                holder.mGender.setImageBitmap(patient.getPhoto());
+            } else {
+                if (patient.getGender().equals("M")) {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsRecyclerViewAdapter.java:105]: Move `"M"` and `"F"` in application constants.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsRecyclerViewAdapter.java [hunk 2]
```diff
@@ -98,7 +99,18 @@ public void onBindViewHolder(@NonNull SyncedPatientsRecyclerViewAdapter.PatientV
...
+                    holder.mGender.setImageResource(R.drawable.patient_male);
+                } else {
+                    holder.mGender.setImageResource(R.drawable.patient_female);
+                }
+            }
+        }
+        if (patient.getDead()) {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/syncedpatients/SyncedPatientsRecyclerViewAdapter.java:112]: Rename the class function to `isDeceased`

File: openmrs-client/src/main/res/values/color.xml [hunk 3]
```diff
@@ -46,6 +46,7 @@
     <color name="dark_grey_6x">#B1B1B1</color>
     <color name="dark_grey_8x">#616161</color>
     <color name="dark_grey_10x">#454545</color>
+    <color name="deceased_red">#F88C8C</color>
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/values/color.xml:49]: Make the red shade lighter.

**Comment by rishabh-997** [comment]
IMG_20200709_001814

**Pull request merged**